tokio/tests/rt_basic.rs

```diff
@@ -16,6 +16,15 @@ mod support {
     pub(crate) mod mpsc_stream;
 }
 
+macro_rules! cfg_metrics {
+    ($($t:tt)*) => {
+        #[cfg(tokio_unstable)]
+        {
+            $( $t )*
+        }
+    }
```

Comment:
Don't we also have a feature flag for this? And why is it not in `src/macros/cfg.rs` ?

Reply:
We do, but tests don't have access to it.